mlflow/pyfunc/__init__.py

```diff
@@ -673,7 +673,7 @@ def add_to_model(
     if model_config:
         params[MODEL_CONFIG] = model_config
     if model_code_path:
-        params[MODEL_CODE_PATH] = model_code_path
+        params[MODEL_CODE_PATH] = os.path.basename(model_code_path)
```

Comment:
This runs when logging the model.
and based on this:
```
import os

path = "/home/user/documents/example.txt"
filename = os.path.basename(path)

print(filename)  # Output: example.txt
```
we don't get the exact path when we load the model back. I am assuming that would be a problem right? 🤔 

Let me know if I am missing something

Reply:
@sunishsheth2009 Good question. When we load a model, we first download the model artifacts (which contains the model script), and then load the model. All we need is a relative path to the script from the model directory, not the original location of the script.

Reply:
> We only need a relative path to the script from the model directory

This is true, but `os.path.basename` only returns the last component name? not including the relative path

Reply:
> os.path.basename only returns the last component name?

Yes, we only need the file name, not the relative path.

Reply:
because we copy the model script in the model directory.

Reply:
Is there any case where the model script is in a subdirectory of the model directory like this?

```
- model
  - scripts
    - model.py
```